cmd/internal/root.go

```diff
@@ -148,9 +147,8 @@ func init() {
 	pf.StringP("bosh-dns-docker-image", "", "coredns/coredns:1.6.3", "The docker image used for emulating bosh DNS (a CoreDNS image)")
 	pf.String("cluster-domain", "cluster.local", "The Kubernetes cluster domain")
 	pf.Int("max-boshdeployment-workers", 1, "Maximum number of workers concurrently running BOSHDeployment controller")
-	pf.Int("max-extendedjob-workers", 1, "Maximum number of workers concurrently running ExtendedJob controller")
-	pf.Int("max-extendedsecret-workers", 5, "Maximum number of workers concurrently running ExtendedSecret controller")
-	pf.Int("max-extendedstatefulset-workers", 1, "Maximum number of workers concurrently running ExtendedStatefulSet controller")
+	pf.Int("max-quarks-secret-workers", 5, "Maximum number of workers concurrently running QuarksSecret controller")
```

Comment:
Is `quarkssecret` not a single word here ?

Reply:
Same as quarks-job, and quarks-statefulset.

Reply:
Okies.